Error when trying to build Renderide on Windows

When trying to build Renderide on Windows, I get the following failure:

```
   Compiling bootstrapper v0.1.0 (C:\Users\bredo\src\tmp\Renderide\crates\bootstrapper)
error[E0599]: no associated item named `Predef` found for struct `RegKey` in the current scope
   --> crates\bootstrapper\src\paths.rs:180:24
    |
180 |     let hklm = RegKey::Predef(HKEY_LOCAL_MACHINE);
    |                        ^^^^^^ associated item not found in `RegKey`
    |
note: if you're trying to build a new `RegKey` consider using one of the following associated functions:
      RegKey::predef
      RegKey::load_app_key
      RegKey::load_app_key_with_flags
   --> C:\Users\bredo\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\winreg-0.56.0\src\reg_key.rs:56:5
    |
 56 |       pub const fn predef(hkey: HKEY) -> RegKey {
    |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
 75 |       pub fn load_app_key<N: AsRef<OsStr>>(filename: N, lock: bool) -> io::Result<RegKey> {
    |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
 99 | /     pub fn load_app_key_with_flags<N: AsRef<OsStr>>(
100 | |         filename: N,
101 | |         perms: Registry::REG_SAM_FLAGS,
102 | |         options: u32,
103 | |     ) -> io::Result<RegKey> {
    | |___________________________^
help: there is an associated function `predef` with a similar name
    |
180 -     let hklm = RegKey::Predef(HKEY_LOCAL_MACHINE);
180 +     let hklm = RegKey::predef(HKEY_LOCAL_MACHINE);
    |

error[E0282]: type annotations needed
   --> crates\bootstrapper\src\paths.rs:183:39
    |
183 |             if let Ok(install_path) = steam_key.get_value::<String, _>("InstallPath") {
    |                                       ^^^^^^^^^ cannot infer type

Some errors have detailed explanations: E0282, E0599.
For more information about an error, try `rustc --explain E0282`.
error: could not compile `bootstrapper` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

Changing `let hklm = RegKey::Predef` to `let hklm = RegKey::predef` at https://github.com/DoubleStyx/Renderide/blob/4fa461d817fc4900903b9125d5799b5ea2bb18aa/crates/bootstrapper/src/paths.rs#L180 appears to fix this error.

--- a/crates/bootstrapper/src/paths.rs
+++ b/crates/bootstrapper/src/paths.rs
@@ -177,7 +177,7 @@ fn steam_path_from_registry() -> Result<PathBuf, std::io::Error> {
     use winreg::enums::HKEY_LOCAL_MACHINE;
     use winreg::RegKey;
 
-    let hklm = RegKey::Predef(HKEY_LOCAL_MACHINE);
+    let hklm = RegKey::predef(HKEY_LOCAL_MACHINE);
     for key_path in &[r"SOFTWARE\WOW6432Node\Valve\Steam", r"SOFTWARE\Valve\Steam"] {
         if let Ok(steam_key) = hklm.open_subkey(key_path) {
             if let Ok(install_path) = steam_key.get_value::<String, _>("InstallPath") {

